### PR TITLE
Adds basic code for creating queue

### PR DESCRIPTION
**Pull request by ROHITCHARDE**
Adds basic code for creating queue

### CODE REVIEW

File: NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs [hunk 1]
```diff
@@ -13,23 +13,84 @@ class Start
...
-            Logger.LogMessage("\nNumber of hours: ");
-            int msgTimeToLiveHours = int.Parse(Console.ReadLine());
-            Logger.LogMessage("\nNumber of minutes: ");
-            int msgTimeToLiveMinutes = int.Parse(Console.ReadLine());
-            Logger.LogMessage("\nNumber of seconds: ");
-            int msgTimeToLiveSeconds = int.Parse(Console.ReadLine());
+            try
+            {
+                //accept required values for creating queue
+                Logger.LogMessage("\nCreating service bus queue......");
```
Comment (rishikeshjadhav) [inline NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs]: Use three dots

File: NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs [hunk 2]
```diff
@@ -13,23 +13,84 @@ class Start
...
+                    if (string.Equals(enableQueuePartitioning, "y", StringComparison.OrdinalIgnoreCase))
+                    {
+                        enablePartitioning = true;
+                    }
+
+                    // Create queue description with entered values                   
+                    QueueDescription queueDescription = new QueueDescription(queueName);
+                    queueDescription.MaxSizeInMegabytes = Convert.ToInt32(maxQueueSize, CultureInfo.InvariantCulture);
+                    queueDescription.DefaultMessageTimeToLive = new TimeSpan(0, 0, 0, Convert.ToInt32(messageTimeToLive, CultureInfo.InvariantCulture));
+                    queueDescription.RequiresDuplicateDetection = Convert.ToBoolean(duplicateDetection, CultureInfo.InvariantCulture);
```
Comment (rishikeshjadhav) [inline NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs]: Remove this

Comment (rishikeshjadhav) [reply NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs]: Remove all

File: NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs (hunk 1 above)
Comment (ROHITCHARDE) [reply NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs]: done

File: NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs (hunk 2 above)
Comment (ROHITCHARDE) [reply NitorOSS.Azure.ProvisionServiceBus/NitorOSS.Azure.ProvisionServiceBus/Start.cs]: done

**Pull request merged**